### PR TITLE
Commented unused modules

### PR DESCRIPTION
**Pull request by muhahahmad68**
Commented unused modules

### CODE REVIEW

File: first_timers/first_timers.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import print_function
 import re
-import warnings
+# import warnings
```
Comment (arshadkazmi42) [inline first_timers/first_timers.py:4]: Good catch. I think we should remove it in place of commenting it out

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Thank you for making this change. I have add a small suggestion once that is fixed we can merge this.

File: first_timers/first_timers.py (hunk 1 above)
Comment (muhahahmad68) [reply first_timers/first_timers.py]: I'll do that shortly

**Review by arshadkazmi42 — APPROVED** [review]
Looks good. Thanks

**Pull request merged**